src/System.Globalization/tests/CharNumericCategory/GetNumericValueTests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), CLRTest.Common.props))\CLRTest.Common.props" />
+  <PropertyGroup>
+    <CLRTestOwner>tasou</CLRTestOwner>
+    <CLRTestSuite>WEEKLY</CLRTestSuite>
```

Comment:
Per Matt's comment and your response, I'm assuming all of these .csproj files will be removed. (And if they were to stay, there are things that would need to be removed, like all of this "CLR"-specific stuff.